gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -295,6 +297,18 @@ struct _GstTensorFilterFramework
    * @param[in/out] private_data A subplugin may save its internal private data here. The subplugin is responsible for alloc/free of this pointer. Normally, close() frees private_data and set NULL.
    */
 
+  int (*invoke_dynamic) (void **private_data, const GstTensorsInfo * in_info, GstTensorsInfo * out_info,
+      const GstTensorMemory * input, GstTensorMemory * output);
+  /**< Mandatory callback. Invoke the given network model.
+   *
+   * @param[in/out] private_data A subplugin may save its internal private data here. The subplugin is responsible for alloc/free of this pointer.
+   * @param[in] info structure of input tensors info
+   * @param[out] info structure of output tensors info. The subplugin should fill this info.
+   * @param[in] input The array of input tensors. Allocated and filled by tensor_filter/main
+   * @param[out] output The array of output tensors. The subplugin should allocate the memory block for output tensor. (data in GstTensorMemory)
+   * @return 0 if OK. non-zero if error.
+   */
+
```

Comment:
The API format looks like v0 format (**private_data). Do not touch v0 format. Update v1 format.
And do not add in the middle of the struct. It breaks ABI of old binaries.

Reply:
I removed `const` from `const prop` in invoke of filter v1 to update in/out tensors info.
If the sub-plugin using filter v0 wants to use a dynamic filter later, it needs to be changed to filter v1.
Thanks.